courses/free-courses-de.md

```diff
@@ -77,6 +91,12 @@
 * [Unreal Engine 4 Tutorial Deutsch/German](https://www.youtube.com/playlist?list=PLNmsVeXQZj7olLCliQ05e6hvEOl6sbBgv) - The Morpheus Tutorials
 
 
+### SQL
+
+* [SQL-Grundlagen](https://wiki.selfhtml.org/wiki/Datenbank/SQL-Grundlagen) - SelfHTML
+* [SQL-Island](https://sql-island.informatik.uni-kl.de) - Spielerisch SQL lernen
```

Comment:
This belongs in a "playgrounds" list

Reply:
@relativvv As an example: https://github.com/EbookFoundation/free-programming-books/blob/main/more/free-programming-playgrounds.md